### PR TITLE
MDP-197: Schedule Filter

### PR DESCRIPTION
**Pull request by slavomircesla**

### CODE REVIEW

File: src/Router.tsx [hunk 1]
```diff
@@ -4,7 +4,6 @@ import { Home } from "./components/Home";
 import { Example } from "./components/Example";
 import { CounterContainer } from "./components/Counter";
 import { Testing } from "./components/Testing";
-
```
Comment (garySiladi) [inline src/Router.tsx:7]: what?

File: src/components/ScheduleFilter/CheckboxLine.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React, { FC, useState } from "react";
+import checkboxEmpty from "../../static/svg/icon-checkbox-empty.svg";
+import checkboxChecked from "../../static/svg/icon-checkbox-checked.svg";
+
+export const CheckboxLine: FC = ({ children }) => {
+  const [image, setImage] = useState(false);
```
Comment (garySiladi) [inline src/components/ScheduleFilter/CheckboxLine.tsx]: rename the values something more suited for the boolean value, something lilke: 
```
const [isOpened, setOpened] = useState(false);
```

File: src/components/ScheduleFilter/CheckboxLine.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+  return (
+    <li
+      className="flex justify-start items-center pl-4 pt-1 text-sm font-lato cursor-pointer"
+      onClick={() => {
+        setImage(!image);
+      }}
+    >
+      {image ? <img src={checkboxChecked} /> : <img src={checkboxEmpty} />}
+      <div>{children}</div>
```
Comment (garySiladi) [inline src/components/ScheduleFilter/CheckboxLine.tsx]: unnecessary div

File: src/components/ScheduleFilter/Dropdown.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import React, { FC } from "react";
+import classnames from "classnames";
+import DropIcon from "../../static/svg/icon-filter.svg";
+
+interface IProps {
+  show: boolean;
+}
+
+export const Dropdown: FC<IProps> = ({ show }) => {
```
Comment (garySiladi) [inline src/components/ScheduleFilter/Dropdown.tsx]: rename `show` to `isShown`

File: src/components/ScheduleFilter/List.tsx [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  "Wine and Paint",
+  "Pet CHAT",
+  "Mandala",
+  "Coaching CHAT",
+  "Skills CHAT",
+  "Indivindi"
+];
+
+export const List = () => {
+  return (
```
Comment (garySiladi) [inline src/components/ScheduleFilter/List.tsx]: return not needed

File: src/components/ScheduleFilter/ScheduleFilter.tsx [hunk 6]
```diff
@@ -0,0 +1,16 @@
+import React, { useState } from "react";
+import { Dropdown } from "./Dropdown";
+import { List } from "./List";
+
+export const ScheduleFilter = () => {
+  const [show, setShow] = useState(false);
```
Comment (garySiladi) [inline src/components/ScheduleFilter/ScheduleFilter.tsx]: rename boolean to something more suiting

**Review by garySiladi — CHANGES_REQUESTED** [review]

**Review by pggl — APPROVED** [review]

**Review by garySiladi — APPROVED** [review]

**Pull request merged**